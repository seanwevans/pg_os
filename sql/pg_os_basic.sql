-- simple regression test
SELECT 1 AS one;
