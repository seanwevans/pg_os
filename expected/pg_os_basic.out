-- simple regression test
SELECT 1 AS one;
 one 
-----
   1
(1 row)

